src/othermbc.md

```diff
@@ -16,22 +16,23 @@ outputs, but this appears not to have been done in licensed games.
 
 ### EMS
 
-PinoBatch learned the game selection protocol for EMS flash carts from
-beware, who in turn learned it from nitro2k01. Take this with a grain of
-salt, as it hasn't been verified on the authentic EMS hardware.
+::: warning
 
-A [header](<#The Cartridge Header>) matching any of the
-following is detected as EMS mapper:
+Take the following with a grain of salt, as it hasn't been verified on authentic EMS hardware.
```

Comment:
We should use a "warning" box with "speculation" or "research needed" as title. This also applies to the other info/warning box around that start with something describing an unverified/speculated behaviour

Reply:
Well, the warning box is already there, only the title needs to be added, then.